configuration/scripts/options/set_nml.boxslotcyl

```diff
@@ -17,6 +18,8 @@ ktherm = -1
 kdyn   = -1
 kridge = -1
 ktransport = 1
+atm_data_type = 'box2001'
+ocn_data_type = 'box2001'
```

Comment:
Or maybe explicitly set them to 'unknown_data_type' if that's available as an option

Reply:
The box2001 are needed for the boxslotcyl.  I think box2001 imposes the solid body rotation forcing?  We use box2001 for a number of other tests where it's a somewhat adhoc choice (I think), but I think it's designed mainly for the boxslotcyl.

Ultimately, we still have this issue of how and where to set namelist options best.  While I think we need a default [atm,ice,ocn]_data_type for each grid, I also think we need to also set it in the other set_nml where appropriate to reinforce the setting.  As we implement the grid_atm and grid_ocn settings, we have similar issues.  It turns out, the forcing wasn't uniformly imposed on the T or U grid for different configurations, so that is also being cleaned up and some of the box results are changing simply because we are now imposing the u/v on the T grid instead of the U grid or similar.  It's been quite a task to test and work all this out.  But we should be in better shape moving forward.